treeherder/etl/push_loader.py

```diff
@@ -86,29 +87,28 @@ def get_info(self):
         return info
 
     def fetch_push(self, url, repository):
-        params = {}
-        params.update(self.CREDENTIALS)
+        params = None
+        # make_request adds the headers but over here we prevent passing the params until
+        # we fully switch over
+        if not self.GITHUB_TOKEN:
+            params = self.CREDENTIALS
 
         logger.info("Fetching push details: %s", url)
 
-        commits = self.get_cleaned_commits(fetch_json(url, params))
+        commits = self.get_cleaned_commits(fetch_json(url, params=params))
         head_commit = commits[-1]
         push = {
             "revision": head_commit["sha"],
+            # A push can be co-authored
+            # The author's date is when the code was committed locally by the author
+            # The committer's date is the info as to when the PR is merged (committed) into master
             "push_timestamp": to_timestamp(
-                head_commit["commit"]["author"]["date"]),
+                head_commit["commit"]["committer"]["date"]
+            ),
+            # We want the original author's email to show up in the UI
             "author": head_commit["commit"]["author"]["email"],
         }
 
-        body = self.message_body
```

Comment:
This code is not needed anymore since we can use `head_commit["commit"]["committer"]["date"]` which is standard for most Github APIs while "timestamp" was *only* available as part of a Pulse event.

See `timestamp` only defined in the [Pulse event](https://github.com/mozilla/treeherder/blob/master/tests/sample_data/pulse_consumer/github_push.json#L166) but "date" for "committer" is not available.